Docs/source/install/hpc/lassen.rst

```diff
@@ -33,9 +33,9 @@ We use the following modules and environments on the system (``$HOME/warpx.profi
    export proj=<yourProject>
 
    # required dependencies
-   module load cmake/3.18.0
+   module load cmake/3.20.2
    module load gcc/8.3.1
-   module load cuda/11.0.2
+   module load cuda/11.2.0
```

Comment:
How come this version of cuda is different than in some of the other file that have it at 11.3?

Reply:
This is only showing up because memaster is out of data with upstream/development. Once we merge that these changes won't show up here anymore. Not sure why upstream changed the cuda version though.